README.MD

```diff
@@ -9,286 +9,91 @@
-
-## Features
-
-***Unstructured text anonymization***
+[Presidio Features](docs/overview.md#features)
 
-Presidio automatically detects Personal-Identifiable Information (PII) in unstructured text, annonymizes it based on one or more anonymization mechanisms, and returns a string with no personal identifiable data.
-For example:
+[Presidio Input and output](docs/overview.md#input-and-output)
 
-[![Image1](docs/assets/before-after.png)](docs/assets/before-after.png)
-
-For each PII entity, presidio returns a confidence score:
-
-[![Image2](docs/assets/findings.png)](docs/assets/findings.png)
-
-***Text anonymization in images*** (beta)
-
-Presidio uses OCR to detect text in images. It further allows the redaction of the text from the original image.
+## Demo
 
-[![Image3](docs/assets/ocr-example.png)](docs/assets/ocr-example.png)
+[Try Presidio with your own data](https://aka.ms/presidio-demo)
 
 ## Learn more
-More information could be found in the [Presidio documentation](https://microsoft.github.io/presidio/).
 
-- [How to deploy presidio](docs/deploy.md)
-- [Setting up a development environment](docs/development.md)
+More information can be found in the [Presidio documentation](https://microsoft.github.io/presidio/).
+
+- [Architecture](docs/design.md)
+- [API Samples](docs/samples.md)
 - [Supported field types](docs/field_types.md)
 - [Database and storage scanner](docs/tutorial_scheduler.md)
-- [Architecture](docs/design.md)
 - [Adding new PII recognizers](docs/custom_fields.md)
 - [Generating Swagger file](presidio-api/cmd/presidio-api/docs/readme.md)
-- [Contributing to Presidio](CONTRIBUTING.md)
 - [Evaluating Presidio](https://github.com/microsoft/presidio-research)
 
+## Deploying Presidio
 
-## Give it a try
-
-* [Run locally with KIND (Kubernetes in Docker)](#run-locally-with-KIND)
-* [Deploy and run on Kubernetes](#single-click-deployment-using-the-default-values)
-* [Install the presidio-analyzer python package](/docs/deploy.md#Install-presidio-analyzer-as-a-Python-package)
+[Deploy Presidio as a system](docs/deploy.md#deploy-presidio-as-a-system):
 
-## Input and output
+- [Install locally using Docker](docs/deploy.md#the-easy-way-with-docker)
+- [Presidio As a Service with Kubernetes](docs/deploy.md#presidio-as-a-service-with-kubernetes)
+- [presidio-analyzer as a standalone python package](docs/deploy.md#install-presidio-analyzer-as-a-python-package)
 
-Presidio accepts multiple sources and targets for data annonymization. Specifically:
+## Developing Presidio
 
-1. Storage solutions
-    * Azure Blob Storage
-    * S3
-    * Google Cloud Storage
-
-2. Databases
-    * MySQL
-    * PostgreSQL
-    * Sql Server
-    * Oracle
-
-3. Streaming platforms
-    * Kafka
-    * Azure Events Hubs
-
-4. REST requests
-
-It then can export the results to file storage, databases or streaming platforms.
+- [Setting up a development environment](docs/development.md)
+- [Run locally with KIND (Kubernetes in Docker)](docs/KIND.md)
```

Comment:
This might need to be in the deploy doc. what do you think?

Reply:
I wanted to seperate between development or test deployment and production deployment. So kept this in a seperate document. Do you suggest combining all into one deployment doc with sub sections for test and production?

Reply:
I think we should go with one of two options:
1. Have all different deployment options (either test or prod) in one file, and reference those that are relevant for development from the develop doc. 
2. Have only the K8S deployment in 'deploy', and the rest under 'develop'

WDYT?

Reply:
After thinking about this I would have one deployment doc with all the options, and seperate them in the doc to "Production" and "Development". LMK if that seems ok and I will updae the doc

Reply:
Sounds good